targets/nano-33-ble.ld

```diff
@@ -0,0 +1,11 @@
+
+MEMORY
+{
+    FLASH_TEXT (rw) : ORIGIN = 0x10000, LENGTH = 1M
```

Comment:
ORIGIN is important, hence separate .ld file

Reply:
I see. However, this is not quite correct: the chip has only 1M of flash and `LENGTH = 1M` will go over this. It should be this:

```suggestion
    FLASH_TEXT (rw) : ORIGIN = 0x10000, LENGTH = 0xf0000
```

Unfortunately, this also means it's probably not possible to use the SoftDevice on this chip without destroying the bootloader.